openshift/release/update-to-head.sh

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
 
-# Synchs the release-next branch to main and then triggers CI
+# Synchs the REPO_BRANCH branch to main and then triggers CI
 # Usage: update-to-head.sh
 
 set -e
 REPO_NAME=$(basename $(git rev-parse --show-toplevel))
+REPO_OWNER_NAME="openshift-knative"
```

Comment:
Is this more of a refactoring?

Reply:
No. It's just a few more lines that are changed to align this with other scripts, e.g. eventing-kafka-broker: https://github.com/openshift-knative/eventing-kafka-broker/blob/main/openshift/release/update-to-head.sh
The main part is changing the "org" in the `hub pull-request` command at the end.